src/main/webapp/assets/js/reviews.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Fetches reviews from backend and displays them in sections under the appropriate business page.
+ */
+async function getReviews() {
+  const response = await fetch('/mockdatareview'); // Fetching mock data for now, to be changed
+  var reviewsList = await response.text();
+
+  /** Display reviews in sections. */
+  reviewsList = JSON.parse(reviewsList)
+  var reviewsContainer = document.getElementById('review-grid');
+  for (i=0; i<reviewsList.length; i++) {
+    var reviewSection = createReviewSection(reviewsList[i]);
+    reviewsContainer.appendChild(reviewSection);
+  }
+}
+
+/**
+ * Returns the formatted review section.
+ */
+function createReviewSection(review) {
+  const reviewWrapper = document.createElement("div");
+  reviewWrapper.class = "review-card";
+
+  reviewWrapper.innerHTML =
+    `<div class="review-card">
+    <div class="review-card-container">
+      <div class="name-and-date">
+        <p>${review.userID}</p>
+        <div class="rating">
```

Comment:
The number of stars should be based on the rating.

I recommend using a js library called underscore!

https://www.geeksforgeeks.org/underscore-js-_-times-function/amp/

Reply:
Thanks! Will settle that in the next PR together with a proper form input for rating

Reply:
Hey @g-lilian ! Is this PR done since you're going to make changes in the next PR? Should I approve and merge?

Reply:
Yes please